### PR TITLE
airflow dashboard

### PR DESCRIPTION
**Pull request by altuner**
Screen Shot 2023-04-11 at 4 22 10 PM

### CODE REVIEW

**Review by nslaughter — CHANGES_REQUESTED** [review]
See comment inline about dashboard_description.
Also, chart titles should be title case.

File: collector-dashboards/otel-collector-airflow-dashboard/main.tf [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+  chart {
+    name = "Scheduler heartbeat"
+    rank = "1"
+    type = "timeseries"
+
+    query {
+      query_name   = "a"
+      display      = "line"
+      hidden       = false
+      query_string = "metric airflow.scheduler_heartbeat | rate | group_by [], sum"
```
Comment (nslaughter) [inline collector-dashboards/otel-collector-airflow-dashboard/main.tf:39]: I don't find this meaningful as a rate. I only care to see heartbeats as a binary status. The squiggles don't help. I'd go so far as to suggest dropping it.

File: collector-dashboards/otel-collector-airflow-dashboard/main.tf [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+  chart {
+    name = "Processes"
+    rank = "0"
+    type = "timeseries"
+
+    query {
+      query_name   = "a"
+      display      = "line"
+      hidden       = false
+      query_string = "metric airflow.dag_processing.processes | rate | group_by [], sum"
```
Comment (hecomp) [inline collector-dashboards/otel-collector-airflow-dashboard/main.tf:25]: Add heredoc here and below `query_string`.

File: collector-dashboards/otel-collector-airflow-dashboard/main.tf [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+    }
+
+  }
+
+  chart {
+    name = "Scheduler heartbeat"
```
Comment (hecomp) [inline collector-dashboards/otel-collector-airflow-dashboard/main.tf:31]: Add all starting caps. Ex:. `Scheduler Heartbeat`. Apply to below chart names.

**Review by hecomp — COMMENTED** [review]
LGTM. Just small comments.

**Review by nslaughter — CHANGES_REQUESTED** [review]
Sorry... didn't notice on my previous that most of these are `rate` and I believe that `delta` is more meaningful. The rate will show us count/interval. To me, the nature of this data is such that the unscaled values are more meaningful.

File: collector-dashboards/otel-collector-airflow-dashboard/main.tf (hunk 2 above)
Comment (hecomp) [reply collector-dashboards/otel-collector-airflow-dashboard/main.tf:25]: > @hecomp are you sure we need it if it perfectly fits in one line?

I believe that Nathan mentioned about keeping consistent. But may be no need.

**Review by nslaughter — APPROVED** [review]

**Pull request merged**